views/userProfile.handlebars

```diff
@@ -9,175 +9,95 @@
                 <div class="card-img-left d-none d-md-flex">
                 </div>
                 <div class="card-body">
-                    <h5 class="card-title text-center">Profile</h5>
-                    <form class="form-signin">
-                        <div class="row">
-                            <div class="col">
-                                {{username}}
-                                {{!-- profile photo  --}}
-                                {{!-- need to add edit, browse, and save for
-                                user to update and change profile pic --}}
-                                <img src="https://mdbootstrap.com/img/Others/documentation/img%20(75)-mini.jpg"
-                                    alt="thumbnail" class="img-thumbnail z-depth-3 rounded float-left"
-                                    style="width: 200px">
-                                {{!-- profile photo  --}}
-                            </div>
-                            <div class="col">
-                                {{!-- Name and contact info (email & phone number) of volunteer  --}}
-                                <div class="card border-info mb-3" style="max-width: 20rem;">
-                                    {{!-- need to make name link to show name displayed from create account page --}}
-                                    <div class="card-header"></div>
-                                    <div class="card-body text-info">
-                                        <div class="text-center">
-                                            {{!-- need to link button to display contact
-                                            info from users create account page --}}
-                                            {{!-- need to code in an edit and save button for 
-                                            users to make changes in contact info --}}
-                                            <a href="" class="btn btn-default btn-rounded mb-4" data-toggle="modal"
-                                                data-target="#modalContactForm">Contact Info</a>
-                                        </div>
-                                    </div>
-                                </div>
-                            </div>
+                    {{!-- <form class="form-signin"> --}}
```

Comment:
get rid of this, it didn't work. keep the left (old) code

Reply:
You want me to delete all the changes you made to this page?

Reply:
The red icon indicates removed files. The green icon indicates added files.